src/lib/simulation.js

```diff
@@ -91,7 +91,7 @@ function addActors(newActors) {
 
 function addActorsFromGui(newActors) {
     newActors.forEach(actor => {
-        const count = actor.count ? actor.count : 1;
+        const count = actor.count;
         for (let i = 0; i < count; i++) {
```

Comment:
doesn't fail if we don't provide a count param for the actor?
how about this?
`const count = actor.count ? actor.count : 0;`

Reply:
actually it will work, count will undefined and i < count will return false. so good as it is